resources/js/views/settings/Application.vue

```diff
@@ -819,6 +819,131 @@
+              <b-input-group v-if="!viewOnly">
+                <b-form-file
+                  id="bbb-logo-form-file"
+                  :state='fieldState("bbb.logo_file")'
+                  :disabled="isBusy || viewOnly || !loaded"
+                  :browse-text="$t('app.browse')"
+                  :placeholder="$t('settings.application.bbb.logo.selectFile')"
+                  v-model="uploadBBBLogoFile"
+                  accept="image/jpeg, image/png, image/gif, image/svg+xml"
+                >
+                </b-form-file>
+                <template #append v-if="uploadBBBLogoFile">
+                  <b-button id="bbb-upload-button" variant="danger" @click="uploadBBBLogoFile = null">
+                    <b-icon-x></b-icon-x>
+                  </b-button>
+                </template>
+              </b-input-group>
+            </b-col>
+          </b-row>
+
+          <template slot='invalid-feedback'>
+            <div v-html="fieldError('logo')"></div>
+            <div v-html="fieldError('logo_file')"></div>
+          </template>
+        </b-form-group>
+
+        <b-form-group
+          label-class="font-weight-bold"
+          class="mb-4"
+          :label="$t('settings.application.bbb.style.title')"
+          label-for='default_presentation'
+          :state='fieldState("bbb.style")'
+        >
+          <b-input-group>
+            <b-form-file
+              accept="text/css,.css"
+              :disabled="isBusy || viewOnly || !loaded"
+              id='default_presentation'
+              :state="fieldState('bbb.style')"
+              :browse-text="$t('app.browse')"
+              :placeholder="$t('settings.application.bbb.style.title')"
+              v-model="bbb_style"
+            >
+            </b-form-file>
+            <b-input-group-append v-if='settings.bbb.style || !!bbb_style'>
+              <!-- View file -->
+              <b-button
+                v-if='settings.bbb.style'
+                variant='dark'
+                :href='settings.bbb.style'
+                target='_blank'
+                ref='view-bbb-style'
+                v-b-tooltip
+                :title="$t('settings.application.bbb.style.view')"
+              >
+                <i class="fas fa-eye"></i>
+              </b-button>
+              <b-button
+                v-if='!viewOnly && (bbb_style !== null || bbb_style_deleted)'
+                variant='secondary'
+                @click='bbb_style = null; bbb_style_deleted = false'
+                ref='reset-bbb-style'
+                v-b-tooltip
+                :title="$t('settings.application.bbb.style.reset')"
+              >
+                <i class="fas fa-undo"></i>
+              </b-button>
+              <!-- Delete file -->
+              <b-button
+                v-if='!viewOnly && bbb_style === null && !bbb_style_deleted'
+                variant='danger'
+                @click="bbb_style_deleted = true"
```

Comment:
Why not just set `bbb_style` to `null`?

Reply:
I think this is something you added for default presentation as well, to revert the deletion before submitting the form

Reply:
Ah, yes.